common/service/dynamicconfig/constants.go

```diff
@@ -760,8 +760,8 @@ const (
 	WorkerReReplicationContextTimeout
 	// WorkerEnableRPCReplication is the feature flag for RPC replication
 	WorkerEnableRPCReplication
-	// WorkerEnableHistoryReplication is the feature flag for history replication
-	WorkerEnableHistoryReplication
+	// WorkerEnableReplication is the feature flag for general replication
+	WorkerEnableReplication
```

Comment:
nit: I can understand the difference between those two configs but it may cause some confusion for the others. Is it possible to update the name to be more specific?

Reply:
Updated